spec/models/custom_value/calculated_value_strategy_spec.rb

```diff
@@ -79,6 +79,14 @@
       end
     end
 
+    context "when value is a float string with many zeroed decimals" do
+      let(:value) { "3.0000000000000000000000000000009" }
```

Comment:
Better to check with few more variants: `2.9…`, `-2.9…`, `-3.0…9`

Reply:
Since I removed the Epsion comparison, I instead added some examples with _no_ and _some_ fractional decimals.